miekmiek/events_lab

```diff
@@ -0,0 +1 @@
+Subproject commit 756e605c2b3acf823e2d12fdd190a6c4dd85a7db
```

Comment:
please move events_lab to your ~/dev/events_lab directory.  Sticking a git repo in a git repo, doesn't work very well.  (Take a look after I merge this).